Doc/library/sqlite3.rst

```diff
@@ -1074,7 +1074,8 @@ Using the connection as a context manager
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Connection objects can be used as context managers
-that automatically commit or rollback transactions.  In the event of an
+that automatically commit or rollback transactions. Note that this does not
+automatically call :meth:`close` on the connection object. In the event of an
```

Comment:
Sorry, by current wording I meant the wording at https://docs.python.org/3/library/sqlite3.html#using-the-connection-as-a-context-manager not in this PR.

Reply:
Sure, I have reverted the commit.

Thanks